### PR TITLE
[onert] Revisit CPU ExternalContext

### PR DESCRIPTION
**Pull request by hseok-oh**
- Introduce thread count getter
- Count physical core on default setting and use it as thread count

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: runtime/onert/backend/cpu/ExternalContext.cc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    }
+  }
+  if (!siblings.empty())
+  {
+    return static_cast<int32_t>(siblings.size());
+  }
+#endif
+
+  unsigned int n_threads = std::thread::hardware_concurrency();
+  return n_threads > 0 ? (n_threads <= 4 ? n_threads : n_threads / 2) : 1;
```
Comment (ragmani) [inline runtime/onert/backend/cpu/ExternalContext.cc:54]: (Out of curiosity)
Can I get the reason why `n_threads / 2` if `n_threads <= 4`? The result may be less than 4.

**Review by ragmani — APPROVED** [review]
LGTM

File: runtime/onert/backend/cpu/ExternalContext.cc (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/backend/cpu/ExternalContext.cc:54]: Maybe it's workaround to use actual physical core count, not thread count.

Comment (ragmani) [reply runtime/onert/backend/cpu/ExternalContext.cc:54]: I see, it may be workaround to use cpu hyper threading.

**Pull request merged**